command/core.py

```diff
@@ -0,0 +1,23 @@
+"""Calls the appropriate handler depending on the event data."""
+from command.commands.user import UserCommand
+
+
+class Core:
+    """Encapsulate methods for handling events."""
+
+    def __init__(self):
+        """Initialize the dictionary of command handlers."""
+        self.__commands = {}
+        self.__commands["user"] = UserCommand()
+
+    def handle_app_mention(self, event_data):
+        """Handle the events associated with mentions of @rocket."""
+        message = (event_data["event"])["text"]
+        s = message.split(' ', 2)
+        if s[0] != "@rocket":
```

Comment:
We'll want to double check if this works once we get the actual Slack app running - I know the RTM API sends mentions with a Slack ID that looks something like `<@u19238>`, but I'm not sure that the Events API does as well. If it does, this line will break.

Reply:
I'm guessing this is something we'll have to look back at later then?

Reply:
Yeah, I don't think we can do it until we actually set it up to receive events from Slack (which I'll... try to get to this week 😂), but it might be helpful to put a TODO up there.